OpenRA.Mods.Common/Widgets/Logic/Ingame/SupportPowerTooltipLogic.cs

```diff
@@ -12,26 +12,31 @@
 using System;
 using OpenRA.Mods.Common.Traits;
 using OpenRA.Primitives;
+using OpenRA.Traits;
 using OpenRA.Widgets;
 
 namespace OpenRA.Mods.Common.Widgets.Logic
 {
 	public class SupportPowerTooltipLogic : ChromeLogic
 	{
 		[ObjectCreator.UseCtor]
-		public SupportPowerTooltipLogic(Widget widget, TooltipContainerWidget tooltipContainer, Func<SupportPowersWidget.SupportPowerIcon> getTooltipIcon, World world)
+		public SupportPowerTooltipLogic(Widget widget, TooltipContainerWidget tooltipContainer, Func<SupportPowersWidget.SupportPowerIcon> getTooltipIcon, World world,
+			Lazy<PlayerResources> playerResources)
 		{
 			widget.IsVisible = () => getTooltipIcon() != null && getTooltipIcon().Power.Info != null;
 			var nameLabel = widget.Get<LabelWidget>("NAME");
 			var hotkeyLabel = widget.Get<LabelWidget>("HOTKEY");
 			var timeLabel = widget.Get<LabelWidget>("TIME");
 			var descLabel = widget.Get<LabelWidget>("DESC");
+			var costLabel = widget.Get<LabelWidget>("COST");
```

Comment:
This feature isn't used in the default mods(?), so not sure if we should make it mandatory. On the other hand, one label more isn't that much overhead.

Reply:
I tried making it optional, but it adds a lot of complications to the code. Also, maps couldn't use the feature if we don't add it here. This will however crash mods after update, which is not nice so the change should be well documented.

Reply:
> so the change should be well documented
Do we have a plan/procedure for this?